juju/client/proxy/kubernetes/proxy.py

```diff
@@ -29,21 +30,18 @@ def __init__(
         self.namespace = namespace
         self.remote_port = remote_port
         self.service = service
+        self.temp_ca_file = None
```

Comment:
Now that this is a filename/path and not a file object, how about we call this `temp_ca_path` to avoid confusion?

Reply:
done.